covid_api/db/static/datasets/agriculture.json

```diff
@@ -3,6 +3,7 @@
     "name": "Agriculture",
     "type": "raster-timeseries",
     "s3_location": "agriculture",
+    "time_unit": "day",
```

Comment:
I think agriculture is monthly

Reply:
![image](https://user-images.githubusercontent.com/11858457/90795055-97c1d800-e2db-11ea-9134-20dee0647567.png)
Date format looks daily (YYYY_MM_DD), but the `CropMonitor` products look like they fall on the 28th of each month

Reply:
ah, let's collapse those to month names only then